synthesizer/abundances.py

```diff
@@ -181,102 +181,172 @@ class Elements:
 
 class Abundances(Elements):
 
-    def __init__(self, Z=Elements.Z_sol, alpha=0.0, CO=0.0, d2m=False, scaling='Dopita+2013'):
+    def __init__(self, Z=Elements.Z_sol, alpha=0.0, C='Dopita2006', N='Dopita2006', d2m=False):
+        
         """
-        This function returns the elemental abundances after removing the depleted amount
+        A class for calcualting elemental abundances including various scaling and depletion on to dust
+
+        Arguments
+        ----------
+        Z : float
+            Mass fraction in metals, default is Solar metallicity.
+        alpha: float 
+            Enhancement of the alpha elements
+        C: float, str
+            log10(C/O) ratio. A str is the name of the function f(Z) to use instead.
+        N: float, str
+            log10(N/O) ratio. A str is the name of the function f(Z) to use instead.
+        d2m: float
+            the fraction of metals in dust
 
-        :param Z: float, the total metallicity (includes depletion as well)
-        :param alpha: float, log10(alpha-enhancement factor)
-        :param CO: float, the abundance of Carbon in CO (not implemented)
-        :param d2m: float(?), dust to metal ratio
-        :param scaling: string
-
-        :return: dictionary with different elemental abundances as log10(element/H)
-        :rtype: float
+        Returns
+        -------
         """
 
-        self.Z = Z
+        # save all parameters to object
+        self.Z = Z # mass fraction in metals
         self.alpha = alpha
-        self.CO = CO
+        self.C = C
+        self.N = N
         self.d2m = d2m
-        self.scaling = scaling
+        
+        # set depletions to be zero
+        self.depletion = {element:0.0 for element in self.all_elements}
 
-        # set all depletions to zero initially
-        self.depletion = {element: 0.0 for element in self.all_elements}
+        # Set helium abundance following Bressan et al. (2012)
+        self.Y = 0.2485 + 1.7756*self.Z
```

Comment:
Does the new Helium calibration from Bressan+2012 assume same solar metallicity as ours?

Reply:
I wondered about this and wasn't sure it should matter. However, it's a very small effect.